code/datums/wounds/pierce.dm

```diff
@@ -106,7 +129,8 @@
 	user.visible_message(span_notice("[user] begins stitching [victim]'s [limb.plaintext_zone] with [I]..."), span_notice("You begin stitching [user == victim ? "your" : "[victim]'s"] [limb.plaintext_zone] with [I]..."))
 	if(!do_after(user, base_treat_time * self_penalty_mult, target=victim, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
 		return
-	user.visible_message(span_green("[user] stitches up some of the bleeding on [victim]."), span_green("You stitch up some of the bleeding on [user == victim ? "yourself" : "[victim]"]."))
+	var/bleeding_wording = (!no_bleeding ? "bleeding" : "holes")
```

Comment:
Please don't invert this if you can switch the message around
```suggestion
	var/bleeding_wording = no_bleeding ? "holes" : "bleeding" 
```

Reply:
fixed